dqm/E3HtmlOutputFile.py

```diff
@@ -69,7 +69,7 @@ def htmlAttributes(**kwargs):
     """
     attributes = ''
     for key, value in kwargs.items():
-        attributes += '%s = %s ' % (key, value)
+        attributes += '%s="%s" ' % (key, value)
```

Comment:
potresti sostituire l'assegnamento, il for e lo strip con questa riga:
`return ' '.join("%s=%s" % item for item in kwargs.items())`

Reply:
Modificato come suggerito (ho aggiunto anche le virgolette attorno al valore per mantenere la modifica che volevo apportare). Normalmente cerco di modificare il meno possibile il codice per rendere la revisione più semplice, ma questo modifica è ovviamente ottima (si evita per esempio lo strip finale).
Colgo l'occasione per augurarle buon Natale *<|:-)
-Luca

Reply:
Ciao Luca! Ottimo!

La filosofia di fare modifiche minime è giustissima. Io in genere faccio molti commit molto piccoli (cercando di esprimere un senso, mica riga per riga :smile:) proprio per facilitare la review  commit per commit.

Grazie per l'impegno che ci metti e buone feste!
Budda